kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.spring.portfolio;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class BoardConfig {
+
+    @Bean
+    Board initialization(){
+        return new Board(toDoTasks(), progressTasks(), endTasks());
+    }
+
+    private TaskList toDoTasks(){
+        return new TaskList();
+    }
+
+    private TaskList progressTasks(){
+        return new TaskList();
+    }
+
+    private TaskList endTasks(){
+        return new TaskList();
+    }
+
+}
```

Comment:
to sie nie zgadza z trescia zadania
```
Napisz klasę konfigurującą Beany o nazwie BoardConfig, która utworzy trzy różne Beany reprezentujące listy zadań i wstrzyknie je do Beana klasy Board.
```
czyli powinny byc lacznie 4 beany `@Bean`. TaskList powinny byc beanami
Board mozesz stworzyc np tak
```
@Bean
public Board board(@Qualifier("toDoTasks") TaskList toDoTasks, @Qualifier("progressTasks") ....

    return new Board(toDoTasks, ...)
```